webserver/Controllers/RealEstatesController.cs

```diff
@@ -27,9 +26,7 @@ public class RealEstatesController : ControllerBase {
             return NotFound();
         }
 
-        var realEstateDto = JsonConvert.SerializeObject( new RealEstateDTO (id, realEstate.Name, realEstate.Address, realEstate.Price, realEstate.CompanyId));
-
-        return Ok(realEstateDto);
+        return Ok(JsonConvert.SerializeObject(realEstate));
```

Comment:
After doing this, you send all the realState fields to the API response.
Is that what you want?

I'm asking it because, usually we create a DTO to avoid leaking data to the API response.

A classic example is the user password in the get user endpoint. But this also applies for other scenarios

Reply:
I figure i'd send the whole Real Estate. There are a few reasons:

- No bandwidth limitation as of now
- All fields are visible for the Clients. There are no "secretive" fields in the Real Estates, no "password" or anything like that
- The only exception being the OwnerCompany, which was ommited from the JSON